plugins/destination/postgresql/pgarrow/to_arrow.go

```diff
@@ -45,6 +45,10 @@ func Pg10ToArrow(t string) arrow.DataType {
 		return arrow.PrimitiveTypes.Int32
 	case "bigint", "int8":
 		return arrow.PrimitiveTypes.Int64
+	case "numeric(20,0)":
```

Comment:
Why not check for `strings.HasPrefix(t, "numeric(") && strings.HasSuffix(t, ",0)")`? (numerics with float components should still remain strings)